scripts/main-built.js

```diff
@@ -300,13 +286,12 @@ async function handleStateSelection() {
               .append($(`<p>${distShortcode}</p>`))
           )
           .append(
-            $('<div class="results-cell election-cell"></div>')
-              .append($(`<p>${electionCode}</p>`))
+            $('<div class="results-cell party-cell"></div>')
+              .append($(`<p>${party}</p>`))
           )
           .append(
             $('<div class="results-cell score-cell"></div>')
                 .append($(`<p>${ccScore}</p>`))
-                .append($(`<p>${electionCode}</p>`))
```

Comment:
I'm going to check with Karan & Caroline, but we might want to actually keep this element here and replace it with the party association, rather than removing it completely, such that rows look like this:
![image](https://user-images.githubusercontent.com/15861748/128761462-ba323c3d-a916-4674-9256-2226db2029b5.png)

instead of this:
![image](https://user-images.githubusercontent.com/15861748/128761544-2919c53a-fed1-495f-be88-59570b06ed62.png)

Since it's a design choice, I want to make sure Karan signs off on it. Otherwise, I'm feeling good about this PR!

Reply:
Checked with Karan, he'd like to use the option where we keep the party affiliation! Just realized I left this comment on the `main-built.js` script, though, so going to go leave a suggested change on the `main.js` script.

Reply:
Ah, didn't realize it was for the mobile view.  I'll add it back in